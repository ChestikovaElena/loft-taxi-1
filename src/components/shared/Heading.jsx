import React from 'react';
import styled from 'styled-components';

const Heading = ({ title }) => <H2>{title}</H2>;

export default Heading;

const H2 = styled.h2`
    margin: 0 0 30px;
    color: #323232;
    font-size: 36px;
    font-weight: 700;
`;
