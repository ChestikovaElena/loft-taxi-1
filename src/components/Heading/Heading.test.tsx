import React from 'react';
import renderer from 'react-test-renderer';
import Heading from './Heading';

describe('Heading', () => {
    it('компонент отображается корректно', () => {
        const headingTree = renderer.create(<Heading title="Заголовок" />).toJSON();

        expect(headingTree).toMatchSnapshot();
    });
});
