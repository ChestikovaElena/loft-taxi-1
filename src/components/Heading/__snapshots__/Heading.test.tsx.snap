// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading компонент отображается корректно 1`] = `
<h2
  className="sc-bdVaJa kALlXQ"
>
  Заголовок
</h2>
`;
